src/sentry/models/projectownership.py

```diff
@@ -22,6 +22,7 @@ class ProjectOwnership(Model):
     raw = models.TextField(null=True)
     schema = JSONField(null=True)
     fallthrough = models.BooleanField(default=True)
+    auto_assignment = models.BooleanField(default=False)
```

Comment:
We can't add this column without being nullable.

Reply:
Nice observation, I see your point, not being nullable it would break existing installations, right?

I think this should be instead handled by the migration though and the specification of a default column value, the default has not been set to `false` and the column is not nullable:
```
sentry=# \d+ sentry_projectownership
                                                          Table "public.sentry_projectownership"
     Column      |           Type           |                              Modifiers                               | Storage  | Stats target | Description 
-----------------+--------------------------+----------------------------------------------------------------------+----------+--------------+-------------
auto_assignment | boolean                  | not null                                                             | plain    |              | 
```

What I am suggesting is that the following `ALTER TABLE` statement should work with existing installations too:

```
ALTER TABLE sentry_projectownership ADD COLUMN new_boolean_col BOOLEAN DEFAULT false;
```

I will check again why this happens and whether it can actually be handled in the migration or not.

Reply:
The issue actually isnt the default as much as it is performance. We can apply live migrations generally when columns are NULLable, as the lock is less restrictive. When doing a non-NULLable column, even one with a default, it has to take a (much) longer lasting lock.

Reply:
Thanks a lot for the detailed explanation @dcramer , I wasn't aware of this limitation!